mandatory/2-function-creation.js

```diff
@@ -16,7 +33,16 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (typeof num === 'number' && num > 100) {
+    return 100 + "%";
+  } else if (num < 100 && num % 1 === 0) {
+    return num + "%"
+  } else if (num % 1 !== 0) {
+    return +num.toFixed(2) + "%";
+  }
```

Comment:
A very interesting solution to detect integer from float 👍
One note: The above conditions won't catch `num = 100` correctly. Can you guess why?